platform/platform-app/src/main/resources/features.xml

```diff
@@ -1050,7 +1050,7 @@
 
     <feature name="platform-all" install="manual" version="${project.version}" hidden="true">
         <feature prerequisite="true">platform-security-session</feature>
-        <feature>platform-migrate-all</feature>
+        <feature prerequisite="true">platform-migrate-all</feature>
```

Comment:
Yeah, I don't know why the migrate stuff would need to wait for the Jetty session stuff. It doesn't use Jetty.